notebooks/leb_salt_evaluation.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            os.environ["HF_TOKEN"] = "xxxxx"
</pre><p>should this be something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">os.environ["HF_TOKEN"] = auth_token
self.auth_token = os.environ.get("HF_TOKEN")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/leb/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>